src/utils.js

```diff
@@ -25,4 +25,33 @@ const parseBearerToken = (rawAuthorizationHeader) => {
   return split[1];
 };
 
-module.exports = { generateSignedJWT, parseBearerToken };
+/**
+ * Get date from 
+ * @param {string} rawAuthorizationHeader
```

Comment:
It looks like this needs to be updated to properly reflect the function as written